experimental/CHANGELOG.md

```diff
@@ -57,8 +59,7 @@ All notable changes to experimental packages in this project will be documented
 
 ### :rocket: (Enhancement)
 
-* Add `resourceDetectors` option to `NodeSDK` [#3210](https://github.com/open-telemetry/opentelemetry-js/issues/3210)
```

Comment:
Is this an expected change?

Reply:
That was an accidental removal during a merge conflict. :( I've put it back now. Sorry about that.